pages/settings/index.tsx

```diff
@@ -97,14 +94,18 @@ const Settings: FC<Props> = (props) => {
 			<Col className='h-full w-full'>
 				<div className='mt-6 w-full rounded-md bg-white p-8 shadow-md dark:bg-section-dark-overlay'>
 					<h3 className='text-xl font-semibold leading-7 tracking-wide text-sidebarBlue dark:text-white'>Settings</h3>
-					<Tabs
-						className='ant-tabs-tab-bg-white font-medium text-sidebarBlue'
-						type='card'
-						defaultActiveKey={tab || 'account'}
-						onTabClick={handleTabClick}
-						items={tabItems}
-						theme={theme}
-					/>
+					{!id ? (
+						<p className='text-white'>Please Login Again</p>
```

Comment:
Please add a link to `login`
and change the text to `Please login to change settings`.